test/cmdlineTests/pretty_json_standard/input.json

```
@@ -0,0 +1,10 @@
+{
+	"language": "Solidity",
+	"sources":
+	{
+		"A":
+		{
+			"content": "// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0; contract C { function f() public pure {} }"
```

You can remove the function to make the test even simpler. We're only concerned with indents after all:
```suggestion
			"content": "// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0; contract C {}"
```
Same in other test input files you added.